src/main/java/uk/gov/hmcts/divorce/solicitor/service/task/SetConfirmServiceState.java

```diff
@@ -19,9 +20,15 @@ public CaseDetails<CaseData, State> apply(final CaseDetails<CaseData, State> cas
 
         CaseData caseData = caseDetails.getData();
 
-        if (!isEmpty(caseData.getApplication().getSolicitorService().getServiceProcessedByProcessServer())) {
-            log.info("Setting State: Holding for CaseId: {}", caseDetails.getId());
-            caseDetails.setState(Holding);
+        if (caseData.getApplicationType().isSole() && caseData.getAcknowledgementOfService().getDateAosSubmitted() != null) {
+            return caseDetails;
```

Comment:
Minor suggestion - add log line mentioning we are skipping moving state as Aos has been submitted previously?

Reply:
Done